setup.py

```diff
@@ -13,17 +13,41 @@
 # limitations under the License.
 
 import os
-import sys
```

Comment:
@tseaver It didn't need to be, but I just did it out of habit since I ripped these out of #2370.